examples/config files - basic/3 connector-ldap.yml

```diff
@@ -30,7 +30,7 @@ credential_manager:
     type: windows_credential_manager
     #service_name is Required for Windows Credential Manger
     service_name: "Internet or Network Address field in Credential Manager"
-    username: "UserName"
+    service_username: "UserName"
```

Comment:
I hope you mean: I demo'd this to the customer today and they did **not** raise any concern on the username part :).

In that case, that's fine, just remove the additional specification of "service_username" from this section and instead use the username field from the original file.  You are given the full config that's in the file as a dictionary, so you can look up the username from there.